client/src/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReviewsContainer from './components/ReviewsContainer.jsx';
+import './index.scss';
+
+const ReviewsService = () => {
+  return (
+    <div>
+      <ReviewsContainer />
+    </div>
```

Comment:
If this `<div>` isn't important to the markup, consider using a React Fragment, the shorthand for which is `<></>`